Cargo.toml

```diff
@@ -18,6 +18,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 [features]
 default = ["proc_macro"]
 proc_macro = ["cached_proc_macro", "cached_proc_macro_types"]
+ahash = ["dep:ahash", "hashbrown/default"]
 async = ["futures", "tokio", "async-trait"]
 async_tokio_rt_multi_thread = ["async", "tokio/rt-multi-thread"]
 redis_store = ["redis", "r2d2", "serde", "serde_json"]
```

Comment:
Can we remove `redis_ahash` feature? (it's a couple of lines below this line)
And instead, if `redis` and `ahash` were enabled, then do the magic?
(I haven't checked where exactly we use `redis_ahash`)

Reply:
I belive we don't directly use `ahash` in the `redis` code it instead allow the `redis` hashmap/set to use ahash to perform faster. https://docs.rs/redis/0.23.3/redis/#optional-features